run_kraken.py

```diff
@@ -22,6 +23,8 @@ def get_node_scenario_object(node_scenario):
         return aws_node_scenarios()
     elif node_scenario['cloud_type'] == 'gcp':
         return gcp_node_scenarios()
+    else:
```

Comment:
Nitpick: I think defaulting to 'generic' if cloud_type is not specified is fine, but let's not allow arbitrary values.   Something like:
if node_scenario['cloud_type'] == 'generic' or not 'cloud_type' in node_scenario:
   generic
else:
  bad cloud type

Reply:
Think that is much better! Added some documentation to show this as well